### PR TITLE
ceph-infra: add new role ceph-infra

### PR DESCRIPTION
**Pull request by guits**
this role manages ceph infra services such as ntp, firewall, ...

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>

### CODE REVIEW

**Comment by leseb** [comment]
Tentative 3.2

File: site.yml.sample [hunk 1]
```diff
@@ -75,6 +75,7 @@
   roles:
     - ceph-defaults
     - ceph-validate
+    - ceph-infra
```
Comment (leseb) [inline site.yml.sample]: would be nice to have tags for each subcomponent such as: 'configure_ntp', 'configure_firewall' so one can skip ntp if they want to.

**Comment by guits** [comment]
jenkins test pipeline

**Comment by guits** [comment]
jenkins test pipeline

**Comment by leseb** [comment]
jenkins test luminous-docker_cluster_collocation

**Comment by guits** [comment]
jenkins test pipeline

**Comment by guits** [comment]
jenkins test pipeline

**Pull request merged**